webapp/tests/i18n_mock.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is this file needed?

Reply:
You're right. This file is not needed. I removed it in the latest commit :+1: